documents/motions/10.2019.2 - Disciplinary Committee.md

```diff
@@ -25,5 +25,7 @@ The WCA Disciplinary Committee is an Advisory Committee of the WCA.
       3. Managing and administrating investigations, reports, decisions, and disciplinary actions.
       4. Keeping all information confidential that needs to stay confidential, based on internal and external guidelines
 2. All WCA Staff must fully cooperate with investigations by the WCA Disciplinary Committee.
-3. All Registered Speedcubers should fully cooperate with investigations by the WCA Disciplinary Committee.
+3. All Registered Speedcubers must fully cooperate with investigations by the WCA Disciplinary Committee.
+   1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee in a one week period.
```

Comment:
```suggestion
   1. All Registered Speedcubers should respond to emails sent on behalf of the WCA Disciplinary Committee within one week of each message.
```

Reply:
I will push a new `Staff review` commit with this right now.